airbyte-server/src/main/java/io/airbyte/server/handlers/ConnectionsHandler.java

```diff
@@ -259,7 +259,7 @@ public ConnectionRead getConnection(final UUID connectionId)
     return buildConnectionRead(connectionId);
   }
 
-  public static CatalogDiff getDiff(final AirbyteCatalog oldCatalog, final AirbyteCatalog newCatalog) {
+  public CatalogDiff getDiff(final AirbyteCatalog oldCatalog, final AirbyteCatalog newCatalog) {
```

Comment:
Why make this non-static?

Reply:
I made this change bc none of the other methods in the ConnectionsHandler are, and it made testing a little odd, since it's mocked and passed in in the tests

Reply:
but let me know if this messes stuff up. as far as I can see, this method is only used in 2 places and only in the web backend connections handler

Reply:
Ah, I hadn't realized it was being mocked in tests when I left this comment. That is a valid reason to make this non-static so that makes sense. Thanks for explaining!